packages/shared/src/components/cards/squad/PlaceholderSquadGrid.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { ComponentProps, ReactElement } from 'react';
+import classNames from 'classnames';
+import { CardTextContainer } from '../Card';
+import { ElementPlaceholder } from '../../ElementPlaceholder';
+
+interface PlaceholderSquadGridProps extends ComponentProps<'div'> {
+  isFeatured?: boolean;
+}
+
+const Text = ({
+  className,
+  ...attrs
+}: ComponentProps<typeof ElementPlaceholder>) => (
+  <ElementPlaceholder
+    {...attrs}
+    className={classNames('h-3.5 rounded-12', className)}
+  />
+);
+
+export const PlaceholderSquadGrid = ({
+  className,
+  isFeatured,
+  ...attrs
+}: PlaceholderSquadGridProps): ReactElement => {
+  const descriptionLength = isFeatured ? 5 : 2;
+  const textLines = Array.from({ length: descriptionLength }).map((_, i) => i);
+
+  return (
+    <div
+      {...attrs}
+      aria-busy
+      className={classNames(
+        'w-80 overflow-hidden rounded-16 border-0 bg-background-subtle p-4',
```

Comment:
I tried to fix this through my phone but needs the laptop, this implementation does not count for few fixes that were done within the categories itself in my last pr. The card itself is not anymore a fixed width. It is given a fixed width on the discover page, while in others it takes the width from the feed container.